pkg/controller/installation/core_controller.go

```diff
@@ -1446,6 +1447,7 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 	} else {
 		instance.Status.ImageSet = imageSet.Name
 	}
+	instance.Status.Version = releaseToBeInstalled
```

Comment:
See my comment on getCalicoVersion, I think this may not be necessary.

Reply:
As discussed, we still need this.